### ISSUE TITLE
Navigation bar

### ISSUE THREAD

**Issue by LadyDeveloper** [chingu-voyages/v9-toucans-team-02#1]
Create a navigation bar:

Home
Moments
Notifications
Messages
Twitter Icon
Search
User Profile
Tweet (Pending of functionality)

**Comment by imusleh** [comment]
I think the home button, profile and the tweet buttons are most important functionalities to work on. The rest we can add as bonus functions??

**Comment by imusleh** [comment]
![nav.png](https://images.zenhubusercontent.com/5cf5a5cb3b5058131162b717/802410cd-7318-4a0a-b6f7-cdcdd541b398)

**Comment by imusleh** [comment]
[ ] Home
 [ ]  Moments
 [ ] Notifications
 [ ] Messages
 [ ] Twitter Bird
 [ ] Search Bar
 [ ] Profile Picture
 [ ] Tweet Button

**Comment by LadyDeveloper** [comment]
I agree. For now we can just use as static/fake link and work in the functionalities later.
